core/extension/gdextension_interface.cpp

```diff
@@ -1484,6 +1485,20 @@ static void *gdextension_classdb_get_class_tag(GDExtensionConstStringNamePtr p_c
 	return class_info ? class_info->class_ptr : nullptr;
 }
 
+GDExtensionObjectPtr gdextension_create_godot_instance(int p_argc, char *p_argv[], GDExtensionInitializationFunction p_init_func) {
+#ifdef LIBRARY_ENABLED
+	return create_godot_instance(p_argc, p_argv, p_init_func);
```

Comment:
I think we probably want to prevent creating more than one of these? Godot uses so many singletons, that I suspect we can't make more than one in the same memory space. (Unless this is somehow supported?)

So, this probably should hang on to the instance somewhere, and error here if we attempt to create another instance? And if we're hanging on to it, it'd probably be good to have another interface function to retrieve it.

Reply:
Yes, this was omitted by accident, we will add it soon.

Reply:
Done.